File: src/ThreadREmQty.java
public class ThreadREmQty extends Thread {
    Stocks s;
    int timing;
    int qty;

    public ThreadREmQty(Stocks s, int timing, int qty) {
        this.s = s;
        this.timing = timing;
        this.qty = qty;
    }

    public void run(){
        
        try {
            s.remQty(qty);
            sleep(timing);

        } catch (InterruptedException e) {
            e.printStackTrace();
        }
    }
    
}

File: pythonVersion/src/ThreadRemQty.py
from Stocks import Stocks
import threading
import time



class ThreadRemQty(threading.Thread):

    def __init__(self, s:Stocks, qty:int, t:int):
        self.s = s
        self.qty = qty
        self.t = t
        
        threading.Thread.__init__(self)
        
    def run(self):
        self.s.remQty(self.qty)
        time.sleep(self.t)